.github/workflows/automation-ci.yml

```diff
@@ -0,0 +1,183 @@
+          REACT_APP_OPENSRP_OAUTH_STATE=opensrp
+          REACT_APP_FHIR_API_BASE_URL=https://fhir.labs.smartregister.org/fhir
+          REACT_APP_OPENSRP_API_V2_BASE_URL=https://opensrp-fhir.labs.smartregister.org/opensrp/rest/
+          REACT_APP_OPENSRP_USER_URL=https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/userinfo
+          REACT_APP_OPENSRP_CLIENT_ID=fhir-core-client
+          REACT_APP_ENABLE_OPENSRP_OAUTH=true
+          REACT_APP_DISABLE_LOGIN_PROTECTION=false
+          REACT_APP_KEYCLOAK_API_BASE_URL=https://keycloak-stage.smartregister.org/auth/admin/realms/FHIR_Android
+          REACT_APP_EXPRESS_OAUTH_LOGOUT_URL=http://localhost:3000/logout
+          REACT_APP_EXPRESS_OAUTH_GET_STATE_URL=http://localhost:3000/oauth/state
+          REACT_APP_KEYCLOAK_LOGOUT_URL=https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/logout
+          REACT_APP_OPENSRP_OAUTH_SCOPES=openid,profile
+          REACT_APP_ENABLE_FHIR_CARE_TEAM=true
+          REACT_APP_KEYCLOAK_USERS_PAGE_SIZE=20
+          REACT_APP_FILTER_BY_PARENT_ID=true
+          REACT_APP_ENABLE_FHIR_HEALTHCARE=true
+          REACT_APP_ENABLE_PLANS=true
+          REACT_APP_ENABLE_LOCATIONS=true
+          REACT_APP_ENABLE_FHIR_LOCATIONS=true
+          REACT_APP_ENABLE_CARD_SUPPORT=true
+          REACT_APP_ENABLE_TEAMS=true
+          REACT_APP_ENABLE_FHIR_PATIENTS=true
+          REACT_APP_ENABLE_TEAMS_ASSIGNMENT_MODULE=true
+          REACT_APP_ENABLE_FORM_CONFIGURATION=true
+          REACT_APP_ENABLE_FHIR_USER_MANAGEMENT=true
+          REACT_APP_LANGUAGE_CODE=en
+          REACT_APP_PROJECT_CODE=core
+          REACT_APP_FHIR_API_BASE_URL=https://fhir.labs.smartregister.org/fhir
+          REACT_APP_ENABLE_FHIR_GROUP=false
+          REACT_APP_ENABLE_FHIR_PRACTITIONER_ROLE=true
+          REACT_APP_PAGINATION_SIZE=1000
+          REACT_APP_ENABLE_SERVER_SETTINGS=false
+          REACT_APP_OPENSRP_WEB_VERSION=$npm_package_version
+          REACT_APP_DEPLOYMENT_ENVIRONMENT=production
+          REACT_APP_ENABLE_FHIR_ORGANIZATION=true
+          REACT_APP_ENABLE_FHIR_TEAMS=true
+          REACT_APP_FHIR_ROOT_LOCATION_IDENTIFIER=eff94f33-c356-4634-8795-d52340706ba9
+          REACT_APP_FHIR_PATIENT_SORT_FIELDS=-_lastUpdated
+          REACT_APP_FHIR_PATIENT_BUNDLE_SIZE=5000
+          REACT_APP_ENABLE_FHIR_HEALTHCARE_SERVICES=false
+          REACT_APP_ENABLE_USER_MANAGEMENT=true
+          REACT_APP_ENABLE_FHIR_COMMODITY=true
+          REACT_APP_ENABLE_QUEST=true
+          EOF
+
+      # cache builds or artifacts, probably cache - TODO
+      - name: Build Application
+        run: |
+          cd web
+          yarn install --frozen-lockfile
+          yarn lerna:prepublish
+
+      - name: Get absolute path for build files
+        run: |
+          echo "REACT_BUILD_FOLDER=$(pwd)/web/app/build" >> $GITHUB_ENV
+
+      - name: Build express application
```

Comment:
@p-netm it seems like it would have been possible/cleaner to run web with docker/docker compose rather than manually setting it up?

Reply:
yeah, thought about it a docker in docker kind of setup, Ill look at doing that in the next iteration. will also create some issues on a few things that can be done better.